roles/senior_engineer.md

```diff
@@ -4,7 +4,7 @@ Our Senior Engineers are crazy about software, have expertise in solving a wide
 
 ## What does the job entail?
 
-We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure on AWS with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a Continuous Delivery pipeline running our automated test suites before deploying changes.
+We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript and C# these days. We will usually be responsible for setting up a customer’s infrastructure on AWS, GCP or Azure with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a CI/CD pipeline running our automated test suites before deploying changes.
```

Comment:
I think this makes it sound like we're C# backend and React/JS frontend nowadays, which isn't quite right, or the message I think we should convey to potential hires. I actually never liked this sentence at all. I'd suggest something such as:

`We build solutions in a variety of languages and platforms, including Ruby on Rails, React, JavaScript and C# - whatever's the right tool for the job.`

Reply:
Yes I like that. makes it more timeless